packages/sdk-components-react/src/components.ts

```diff
@@ -17,6 +17,7 @@ export { Subscript } from "./subscript";
 export { Button } from "./button";
 export { Input } from "./input";
 export { Form } from "./form";
+export { Form as RemixForm } from "./form";
```

Comment:
can we rename that form to remix form?

Reply:
It's not remix form 😄
We just alias to it for compatibility